en/sql/tuning.rst

```diff
@@ -2337,3 +2340,80 @@ The above **SELECT** query's plan is printed out as below; we can see "(sort lim
+
+    csql> SELECT /*+ QUERY_CACHE */ count(*) FROM game;
+
+    === <Result of SELECT Command in Line 1> ===
+
+         count(*)
+    =============
+         8653
+
+    1 row selected. (0.107082 sec) Committed.
+
+    1 command(s) successfully processed.
+
+    csql> SELECT /*+ QUERY_CACHE */ count(*) FROM game;
+
+    === <Result of SELECT Command in Line 1> ===
+
+         count(*)
+    =============
+         8653
+
+    1 row selected. (0.003932 sec) Committed.
+
+    1 command(s) successfully processed.
+
+The user can check the query to be cached or not by putting the session command *;info qcache'* in CSQL as follows: ::
+
+    csql> ;info qcache
+
+    LIST_CACHE {
+      n_hts 1010
+      n_entries 1  n_pages 1
+      lookup_counter 1
+      hit_counter 1
+      miss_counter 0
+      full_counter 0
+    }
+
+    list_hts[0] 0x6a74d10
+    HTABLE NAME = list file cache (DB_VALUE list), SIZE = 211, REHASH_AT = 147,
+    NENTRIES = 1, NPREALLOC = 0, NCOLLISIONS = 0
+
+    HASH AT 0
+    LIST_CACHE_ENTRY (0x6c46d18) {
+      param_values = [ ]
+      list_id = { type_list { 1 integer/1 } tuple_cnt 1 page_cnt 1 first_vpid { 65 32766 } last_vpid { 65 32766 } lasttpl_len 24 query_id 2
+      temp_vfid { 64 32766 } }
+      uncommitted_marker = false
+      tran_isolation = 4
+      tran_index_array = [ ]
+      last_ta_idx = 0
+      query_string = select /*+ QUERY_CACHE */ count(*) from [game] [game]?193="en_US";194="en_US";249="Asia/Seoul";user=0|833|1
+      time_created = 11/23/20 16:07:12.779703
+      time_last_used = 11/23/20 16:07:22.772330
+      ref_count = 1
+      deletion_marker = false
+    }
+
+The cached query is shown as **query_string** at the middle of the result screen. Each of the **n_entries** and **n_pages** represents the number of cached query and the number of pages in the cached results. The **n_entries** is limited to the value of configuration parameter **max_query_cache_entries** and the **n_pages** is limited to the value of **query_cache_use_pages**. If the **n_entries** is overflown or the **n_pages** is overflown, some victims among the cache entries are selected and they are uncached. The number of victims is about 20% of **max_query_cache_entries** value and of the **query_cache_use_pages** value.
```

Comment:
1. at the middle of the result screen--> in the middle ('at' change to 'in')

2. the number of cached query-->the number of cached queries ( plural)

Reply:
Ok, thanks.